data.sql

```diff
@@ -84,7 +84,14 @@ INSERT INTO visits (animal_id, vet_id, date) VALUES (1,1,'2020-05-24'),
 
 
 
-
+INSERT INTO visits (animal_id, vet_id, date)
```

Comment:
- [ ] It is a good practice to ensure you have a valid license properly set up on your projects. The MIT License link on your README file links to a **404** page. Kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE
https://choosealicense.com/licenses/mit/